### DISCUSSION TITLE
(Feature request) Home-assistant addon

### DISCUSSION THREAD

**Discussion by HyperCriSiS** [TriliumNext/Notes#735] [Ideas]
### Describe feature

Hi,
Would be awesome to have TriliumNext as Home-Assistant addon.
As the addons are also docker containers, I think this could be done relatively uncomplicated.
It would attract a large user base.

Best regards
Hyper

**Comment by meichthys** [comment]
This sounds neat, but probably not something we'd want to take on as part of the core project. 

We would however encourage anyone to create a third-party home-assistant addon. If it gains traction and is well-maintained, we'd be glad to feature a home-assistant badge on our readme and link to the add-on 👍

**Comment by CobriMediaJulien** [comment]
@meichthys should we close this one and port it to the issues/ feature requests to maintain the discussion cleaner?

we can track this there, also i think it´s low prio right now.

**Comment by meichthys** [comment]
I woulndt be against it, but I would prefer to leave this as a discussion since it would not be a feature of the Notes application, but rather a separate way to host the application from within Home Assistant.

I don't really think this is something we should take on as part of the core project, but rather a third party could implement this and keep it up to date with our project here.

**Reply by CobriMediaJulien** [reply]
you are completely right.

**Comment by HyperCriSiS** [comment]
https://github.com/orgs/TriliumNext/discussions/1403
